docs/source/guides/dm.md

```diff
@@ -24,7 +24,7 @@ function.
 Here's an example:
 
 ```py
-from instrument.utils.aps_functions import aps_dm_setup
+from apstools.utils.aps_functions import aps_dm_setup
```

Comment:
This instruction is an error.  There is no such code in [apstools](https://github.com/search?q=repo%3ABCDA-APS%2Fapstools%20aps_dm_setup&type=code).  Please check these libraries.